packages/react-composites/src/composites/localization/locales/en-US/strings.json

```diff
@@ -68,7 +68,10 @@
     "holdScreenLabel": "You're on hold",
     "openDtmfDialpadLabel": "Show dialpad",
     "dtmfDialpadPlaceholderText": "Enter number",
-    "outboundCallingNoticeString": "Calling..."
+    "outboundCallingNoticeString": "Calling...",
+    "participantJoinedNoticeString": "{displayName} joined",
+    "participantLeftNoticeString": "{displayName} left",
+    "defaultParticipantChangedString": "participant"
```

Comment:
consider 'unnamed participant' to align with other places that have default text for participants without names: https://github.com/Azure/communication-ui-library/pull/1978/files#diff-3a3f9e0f0171ef781ad482c272de2534023eb9b9449a5dca3fdbe3f13ee04df6